### DISCUSSION TITLE
After Retiring the user, Can we unretire that user?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#649] [General Discussion]
We are using the 4.6 version of Sunbird.
We want to retire some users but don't know if we can unretire those users in the future when we need them.
Can we unretire the retired users? if yes then how.
Do we have any API like we have retired user API?
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by Hari-stackroute** [comment]
@sachinb80 there is no retire concept for users, we can block and unblock the users with respective api's
You can find the details in https://lern.sunbird.org/use/developer-guide/user-and-org-service/apis/user-management
`https://staging.open-sunbird.org/api/user/v1/block`
`https://staging.open-sunbird.org/api/user/v1/block`
